src/common/translation/i18n/fi.json

```diff
@@ -516,8 +516,8 @@
             "error": {
               "length": "Postinumeron täytyy olla 5 merkkiä pitkä"
             },
-            "label": "Lapsen postinumero",
-            "placeholder": "Lisää lapsen postinumero"
+            "label": "Postinumero",
+            "placeholder": "Lisää postinumero"
```

Comment:
This change affects `e2e-test selector`

 TestingLibraryElementError: Unable to find a label with the text
      of: Lapsen postinumero*

Reply:
Fixed 👍